command.go

```diff
@@ -253,6 +253,7 @@ func parseEnvArgs(args []string) (Command, error) {
 	flag := pflag.NewFlagSet("env", pflag.ContinueOnError)
 	flag.String("format", "shell", "Specify what built in format to output variables in (shell, sh, fish, json) or a text template. Default: shell")
 	flag.String("assume", "", "Role to assume")
+	flag.Bool("refresh", false, "Refresh session expiration")
```

Comment:
This makes it sound like we're specifically extending the expiration of the session, which isn't true at all. Instead, we're ignoring any existing session, generating a new session with new temporary credentials. The help text should reflect as much.